framework/include/utils/SolutionInvalidity.h

```diff
@@ -97,12 +110,21 @@ class SolutionInvalidity : protected ConsoleStreamInterface, public libMesh::Par
    */
   void print(const ConsoleStream & console) const;
 
+  /**
+   * Print the time history table of Solution Invalid warnings
+   * @param console The output stream to output to
+   */
+  void printHistory(const ConsoleStream & console, unsigned int & time_interval) const;
```

Comment:
```suggestion
  void printHistory(const ConsoleStream & console, unsigned int & timestep_interval_size) const;
```
time_interval a tad less clear